api-reference/v1.0/resources/groups_overview.md

```diff
@@ -0,0 +1,126 @@
+    "groupTypes": [
+        "Unified",
+        "DynamicMembership"
+    ],
+    "mailEnabled": true,
+    "mailNickname": "marketing",
+    "securityEnabled": false,
+    "membershipRule": 'user.department -eq "Marketing"',
+    "membershipRuleProcessingState": "on"
+}
+```
+
+Learn more about formulating membershipRules in [advanced rules](https://docs.microsoft.com/en-us/azure/active-directory/active-directory-groups-dynamic-membership-azure-portal).
+
+**NOTE** Dynamic membership rules requires the tenant to have a license at tier [Azure Active Directory Premium P1](https://azure.microsoft.com/en-us/pricing/details/active-directory/) or greater.
+
+## Common use cases
+
+Using Microsoft Graph, you can perform the these common operations and more:
+
+| **Use cases**		   | **REST resources**	| **See also** |
+|:---------------|:--------|:----------|
+| **Group object and methods** | | |
+| Create new groups, get existing groups, update the properties on groups, and delete groups. Currently, only security groups and Outlook groups can be created through the API. | [group](group.md) | [Create new groups](../api/group_post_groups.md) <br/> [List groups](../api/group_list.md) <br/> [Update groups](../api/group_update.md) <br/> [Delete groups](../api/group_delete.md) |
+| **Group membership methods** | | |
+| List the members of a group, and add or remove members. | [user](user.md) <br/> [group](group.md)| [List members](../api/group_list_members.md) <br/> [Add member](../api/group_post_members.md) <br/> [Remove member](../api/group_delete_members.md)|
+| Check if a user is a member of a group, get all the groups the user is a member of. | [user](user.md) <br/> [group](group.md)| [Check member groups](../api/group_checkmembergroups.md) <br/> [Get member groups](../api/group_get_membergroups.md)|
+| List the owners of a group, and add or remove owners. | [user](user.md) <br/> [group](group.md)| [List owners](../api/group_list_members.md) <br/> [Add member](../api/group_post_members.md) <br/> [Remove member](../api/group_delete_members.md)|
+
+## Other types of groups
+
+You may encounter some of these types of groups being returned when performing operations that return collections of groups. The following groups can be returned through Microsoft Graph, but can't be created or otherwise managed using this API.
+
+| Type              | Use case | groupType | mail-enabled | security-enabled | Learn more |
+|-------------------|----------|-----------|--------------|------------------|------------|
+| Groups in Yammer | Facilitating user collaboration with through Yammer posts. | ["Unified"] | true | false | [Yammer developer API docs](https://developer.yammer.com/docs) |
+| Mail-enabled security groups | Controlling user access to in-app resources, with a shared group mailbox. | [] | true | true | [Manage mail-enabled security groups Exchange article](https://technet.microsoft.com/en-us/library/bb123521(v=exchg.160).aspx) |

```

Comment:
I checked with Yammer. You actually can check conversations. It might be empty, but it exists. So I've found no way to see if a group is a Yammer or Outlook group unless I go to /beta and GET the group, and see "resourceProvisioningOptions" = "YammerProvisioning"

I can fold the mail-enabled SG into the top table, but I don't think we should fold Yammer groups up there. You can't even get the posts in a Yammer group in the API, nor tell that this is a Yammer group. I'll move Mail-enabled SG up though.